tests/behaviour/concept/thing/relation/relation_steps.py

```diff
@@ -87,51 +86,52 @@ def step_impl(context: Context, var1: str, role_label: str, var2: str):
 def adding_player_throws_exception(context: Context, var1: str, role_label: str, var2: str):
     relation = context.get(var1).as_relation()
     try:
-        relation.as_remote(context.tx()).add_player(
-            relation.get_type().as_remote(context.tx()).get_relates(role_label),
+        relation.add_player(
+            context.tx(),
+            relation.get_type().get_relates(context.tx(), role_label),
             context.get(var2))
         assert False;
-    except TypeDBClientException:
+    except (TypeDBClientException, RuntimeError):
```

Comment:
Can this be more specific than `RuntimeError`?

Reply:
There is no more specific exceptions in SWIG's list of possible python exceptions.